File: Coding Ninjas - CodeStudio/Basics Of C++ Guided Path/fahrenheit_to_celsius.cpp
#include <iostream>
#include <cmath>
using namespace std;

int main() {
	int s, e, w;
    cin >> s;
    cin >> e;
    cin >> w;
    for(int i = s; i <= e; i += w) {
        float c = (i - 32) * 5 / 9;
        if(c < 0)
        	cout << i << " " << floor(c) << endl;
        else
            cout << i << " " << ceil(c) << endl;
    }
    return 0;
}

File: Coding Ninjas - CodeStudio/Basics of Java Guided Path/fahrenheit_to_celsius.java
import java.util.Scanner;
class Solution {
	
	public static void main(String args[]) {
		
		Scanner sc = new Scanner(System.in);
        
        int s = sc.nextInt();
        int e = sc.nextInt();
        int w = sc.nextInt();
        
        for(int f = s; f <= e; f += w) {
            float c = (float)(5.0 / 9) * (f - 32);
            
            if(c < 0)
            	System.out.println(f + "\t" + (int)Math.ceil(c));
            else
                System.out.println(f + "\t" + (int)Math.floor(c));
        }
        
	}
}